.internal-ci/helm/mc-core-dev-env-setup/values.yaml

```diff
@@ -110,6 +114,7 @@ consensusNodeConfig5:
       hostname: '{{ printf "node5.%s.development.mobilecoin.com" .Release.Namespace }}'
     peer:
       hostname: '{{ printf "peer5.%s.development.mobilecoin.com" .Release.Namespace }}'
+    txSourceUrl: '{{ with .Values.global.node.ledgerDistribution }}{{ printf "https://s3-%s.amazonaws.com/%s/%s/" .awsRegion .s3Bucket (printf "node5.%s.development.mobilecoin.com" $.Release.Namespace) }}{{ end }}'
```

Comment:
This pattern seems like a good candidate for a helper.

Reply:
That chart is basically one big helper.   The template resolution is done in the child charts.